app/common/state/pageDataState.js

```diff
@@ -52,6 +52,28 @@ const pageDataState = {
     return state.setIn(['pageData', 'last', 'tabId'], tabId)
   },
 
+  saveLastClosedTab: (state, tabValue) => {
+    if (tabValue == null) {
+      return state
+    }
+
+    return state.setIn(['pageData', 'last', 'closedTabValue'], tabValue)
```

Comment:
I think this is going to cause a privacy problem for incognito mode

Reply:
fixed